measures/ResStockArgumentsPostHPXML/measure.rb

```diff
@@ -956,16 +959,50 @@ def set_electric_panel(runner, hpxml_bldg_existing, hpxml_bldg, args)
                           component_idrefs: [ev_charger.id])
     end
 
-    if !electric_panel_load_other_power_rating.nil? || !args[:electric_panel_load_other_new_load].nil?
-      branch_circuits.add(id: "BranchCircuit#{branch_circuits.size + 1}",
-                          occupied_spaces: 1,
-                          component_idrefs: [])
-      service_feeders.add(id: "ServiceFeeder#{service_feeders.size + 1}",
-                          type: HPXML::ElectricPanelLoadTypeOther,
-                          power: electric_panel_load_other_power_rating,
-                          is_new_load: args[:electric_panel_load_other_new_load],
-                          component_idrefs: [])
-    end
+    branch_circuits.add(id: "BranchCircuit#{branch_circuits.size + 1}",
+                        occupied_spaces: 1,
+                        component_idrefs: [])
+    service_feeders.add(id: "ServiceFeeder#{service_feeders.size + 1}",
+                        type: HPXML::ElectricPanelLoadTypeOther,
+                        power: electric_panel_load_other_power_rating,
+                        is_new_load: args[:electric_panel_load_other_new_load],
+                        component_idrefs: [])
```

Comment:
Pretty sure `electric_panel_load_other_power_rating` is never nil, so we can just always add a branch circuit and service feeder.